code/modules/clothing/glasses/glasses.dm

```diff
@@ -511,6 +513,16 @@ var/list/science_goggles_wearers = list()
 				M.disabilities &= ~NEARSIGHTED
 	..()
 
+/obj/item/clothing/glasses/thermal/equipped(mob/user)
+	seedarkness = FALSE
+	..()
+
+/obj/item/clothing/glasses/thermal/unequipped(mob/user, var/from_slot = null)
+	if(from_slot == slot_glasses)
+		seedarkness = TRUE
+	..()
+
+
```

Comment:
I don't get it. It only has `seedarkness = TRUE` when not equipped? Does that do anything?

Reply:
That's a pretty good question, it would appear it doesn't.